docker/docker.go

```diff
@@ -11,16 +11,15 @@ const (
 	Interval = 5 * time.Second
 )
 
-type DockerClient struct {
+const SRV_NAME = "SRV_NAME"
+
+type ConcreteDockerClient struct {
```

Comment:
Just my personal opinion, but it feels a bit over engineered. Since the docker.Client is not an interface we're wrapping it around with another client, so we always have to call `client.client`. I think I would just stub the whole thing with something like this: https://github.com/fsouza/go-dockerclient/blob/master/testing/server.go
Maybe it would be a bit less correct, but it can make the code easier to understand and the tests will be more descriptive.